src/test/java/org/jitsi/meet/test/web/WebTestBase.java

```diff
@@ -258,6 +301,23 @@ private Participant joinParticipant(
      */
     public Participant joinFirstParticipant()
     {
-        return joinParticipant(0, null, null, null);
+        return joinParticipant(0, null, null);
+    }
+
+    /**
+     * Return new {@link JitsiMeetUrl} instance which has only
+     * {@link JitsiMeetUrl#serverUrl} field initialized with the value from
+     * {@link #JITSI_MEET_URL_PROP} system property.
+     *
+     * @return a new instance of {@link JitsiMeetUrl}.
+     */
+    @Override
```

Comment:
If we'd want to append default config for mobile we'll have to override this method and duplicate the logic. If default config would be a param to base class then it could do it. Anyway it probably fits better to Participant, because a test may want to have both mobile and web participants. Plus this config part is specific to web/mobile participant and it feels like it belongs to Participant.